cmp/acdf40a4-bde1-4006-93ec-4e985848c10c/component.lp

```diff
@@ -0,0 +1,65 @@
+(librepcb_component acdf40a4-bde1-4006-93ec-4e985848c10c
+ (name "Nixie Tube Without Dot")
+ (description "A nixie tube with 10 digits but no dot.")
+ (keywords "nixie,tube")
+ (author "Danilo Bargen")
+ (version "0.1")
+ (created 2019-09-16T22:18:58Z)
+ (deprecated false)
+ (category 0d3a9561-9a14-4622-98b2-d787b8b7478c)
+ (schematic_only false)
+ (default_value "{{ PARTNUMBER or DEVICE or COMPONENT }}")
```

Comment:
Since it should be clear from the symbol that it's a nixie tube, I'd remove "or COMPONENT". If you agree:

```suggestion
 (default_value "{{ PARTNUMBER or DEVICE }}")
```